src/Model/Item.php

```diff
@@ -2211,8 +2274,13 @@ public static function getLanguageMessage(array $item): string
 		foreach (json_decode($item['language'], true) as $language => $reliability) {
 			$code = DI::l10n()->toISO6391($language);
 
-			$native   = $iso639->nativeByCode1($code);
-			$language = $iso639->languageByCode1($code);
+			if ($code == 'un') {
```

Comment:
Other location for the `UNDETERMINED_LANGUAGE` constant.

Reply:
Done